crates/orchestrator/src/jobs/da_job/mod.rs

```diff
@@ -62,6 +62,7 @@ pub struct DaJob;
 
 #[async_trait]
 impl Job for DaJob {
+    #[tracing::instrument(name = "da_create_job", skip(self, _config, metadata))]
```

Comment:
Since there are 6 instances of 
- create_job 
- process_job
- verify_job

( General + SNOS, Proving, Proof Registry, DA, State Update )
Can we have two identifiers :
`type` and `category`
type defined whether it's a `create` | `process` | `verify`
category defines whether it's a `general`| `snos` | `proving` | `proof_registry` | `da` | `state_update`

WDYT @apoorvsadana  ?

Reply:
do we need `type`? we will get that from the function name itself right?

Reply:
Given the function name is similar across all the implementations, you are right we can work without type, 
[here's the implementation, check it out](https://github.com/madara-alliance/madara-orchestrator/pull/131/commits/174b64dcfa4c0dbdba656301d9191550ffc20653).
It is to note that `snos` and `proof_registry` are not instrumented since they are not implemented right now.

Reply:
the functions are there right, let's add instrument macro still, implementations will update with time

Reply:
Resolved [here](https://github.com/madara-alliance/madara-orchestrator/pull/131/commits/8ba02e6ca1cf624871bd4646c4e05d7364476bc2).